packages/arb-token-bridge-ui/src/util/wagmi/setup.ts

```diff
@@ -124,7 +124,11 @@ export function getProps(targetChainKey: string | null) {
     // https://github.com/wagmi-dev/references/blob/main/packages/connectors/src/walletConnect.ts#L114
     getChains(sanitizeTargetChainKey(targetChainKey)),
     [
-      infuraProvider({ apiKey: process.env.NEXT_PUBLIC_INFURA_KEY! }),
+      infuraProvider({
+        apiKey:
```

Comment:
I don't think this will work, as it expects same key will be used for all chains